geth/node/manager_test.go

```diff
@@ -118,14 +118,14 @@ func (s *ManagerTestSuite) TestReferences() {
 			func() (interface{}, error) {
 				return s.NodeManager.RPCClient(), nil
 			},
-			node.ErrNoRunningNode,
+			nil,
```

Comment:
This can lead to segfault errors as nothing checks for the rpc client being `nil`. All other getters return `(obj, error)`. Just wanted all of us to be aware of that, I don't think we need to change this.

Reply:
Sorry, didn't get it. Could you elaborate?

Reply:
This test is suppose to verify `"non-null manager, no running node, get RPC Client",`. In such a case, we usually return `error` for most of the getters. But here, `s.NodeManager.RPCClient()` returns `nil` and we don't check that. If `nil` is not checked and the code can try to use the rpc client and when a method is called on `nil`, it's segfault.